x-pack/plugins/reporting/server/lib/tasks/execute_report.ts

```diff
@@ -126,15 +140,17 @@ export class ExecuteReportTask implements ReportingTask {
     const maxAttempts = task.max_attempts;
     if (report.attempts >= maxAttempts) {
       const err = new Error(`Max attempts reached (${maxAttempts}). Queue timeout reached.`);
-      await this._failJob(task, err);
+      const resp = await this._failJob(report, err);
+      report._seq_no = resp._seq_no;
+      report._primary_term = resp._primary_term;
```

Comment:
Hm, it already has been "saved" via calling `await this._failJob(report, err)`. It looks like we should not update `report` after this last update on the document - there is nothing we can do with the updated info since that error is thrown after the "failure" update. 

I removed the lines in this commit: 5ce7541294e 

This manual test covers this code and still passes:
 - In kibana.yml, set `xpack.reporting.capture.maxAttempts: 1` (default in dev mode)
 - In kibana.yml, set `xpack.reporting.queue.timeout: '20s'`
 - Queue a PDF report
 - While the report is executing, restart the dev server (open a source file and save)
 - When the server restarts, the report will be found by the monitoring task and re-scheduled (within the configured 20 seconds)
 - When Task Manager distributes the execute task, instead of claiming the task, it is immediately failed because there is only 1 attempt allowed.